pkg/odo/cli/pipelines/utility/utility.go

```diff
@@ -0,0 +1,16 @@
+package utility
+
+import (
+	"strings"
+
+	"github.com/openshift/odo/pkg/log"
+)
+
+// AddGitSuffix will append .git to URL
+func AddGitSuffix(url string) string {
+	if url == "" || strings.HasSuffix(url, ".git") {
```

Comment:
What if I have URLs like "https://github.com/foo/bar.GIT" or "https://github.com/foo/bar.Git"?   They are both valid URLs as URLs are case insensitive.

Reply:
I missed this corner case. I will fix it. Thanks William!

Reply:
This case is handled now